cypress/features/regression/buttonToggleGroup.feature

```diff
@@ -105,10 +104,22 @@ Feature: Button Toggle Group component
 
   @positive
   Scenario Outline: Change Toggle Button Group label align to <direction>
-    When I check labelInline checkbox
-      And I select labelAlign to "<direction>"
+    Given I check labelInline checkbox
+    When I select labelAlign to "<direction>"
     Then label Align on preview is "<direction>"
     Examples:
       | direction |
       | left      |
-      | right     |
\ No newline at end of file
+      | right     |
+
+  @positive
+  Scenario Outline: Verify the onChange event for a Button Toggle Group <button>
+    Given clear all actions in Actions Tab
+    When I click on Button Toggle Group "<button>"
+    Then <action> action was called in Actions Tab
+    Examples:
+      | button | action   |
+      | Foo    | onChange |
+  # @ignore until component default state of selected is removed for FE-2346
+  # | Bar    | onChange |
+      | Baz    | onChange |
```

Comment:
the same here we should remove the action from Examples and put it directly in Then step if it has no difference :)

Reply:
Amended 👍